.changelog/3283.internal.md

```diff
@@ -0,0 +1 @@
+go/ci: Update nancy
```

Comment:
Maybe specify to which version it has been updated.

Reply:
will do